src/services/message.service.ts

```diff
@@ -31,6 +31,46 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendMessage(
+    message: IMessage,
+    content: string,
+    reply: boolean,
+    argHeader?: string,
+    argFooter?: string,
+    argBreakChar?: string
+  ): Promise<Boolean> {
+    const header = argHeader ? argHeader + '\n' : '';
+    const footer = argFooter ? '\n' + argFooter : '';
+    const breakChar = argBreakChar || '';
+
+    const replyLength = reply ? ('<@' + message.member?.id + '>,  ').length : 0;
+    const miscLength = header.length + footer.length;
```

Comment:
i think we can make this name a little more insightful. `templateLength` might give off more signal to what the relevance of this variable is